protocol/release_management.md

```diff
@@ -0,0 +1,60 @@
+Summary
+-------
+
+Overiew of the release management process for all software applications.
+
+## Release and QC
+1. Development Manager will assign a developer as Release Manager for the
+   application.
+2. Development Manager will create a JIRA ticket for the release and assign to
+   the Release Manager.
+3. Release Manager will navigate to develop branch
+```
+git checkout develop
+```
+4. Release Manager will create a release branch using the proper version for the
+   branch name.
+```
+git flow release start 1.2.3
+```
+5. For projects where there is an application version file, update to match the
+   release version and commit the file to the release branch.
+6. Publish the local release branch to Github
+```
+git flow release publish 1.2.3
+```
+7. Move the staging tag to point to the release branch and push tag to Github
+```
+git tag -a -f staging -m "release 1.2.3"
+git push -f --tags
+```
+8. Bamboo CI will detect the release/1.2.3 branch and will run test suite and
+   static analysis tooling via Code Climate. If successful, application will be
+deployed to the test environment. If not, Release Manager is responsible for
+fixing reported errors.
+9. Release Manager will assign the Deploy ticket to the Product Owner for
+   review.
+10. If problems are detected:
+  1. The Development Team will make needed code changes on the release branch
+  2. Commit and push changes to the remote branch
+  3. Write additional tests, if necessary, to cover the problem space detected
+  4. Run application test suite and Code Climate to verify all tests and
+     analysis tools pass.
+11. When Product Owner signs off on release, they will assign to Release
+    Manager.
+
+## Production
+1. When Product Owner has assigned the Deploy ticket back to the Release
+   Manager, the release branch must be finished prior to deployment.
+```
```

Comment:
We should add fetching master with `git fetch master` -- if someone else did the last release, you might not have the current master branch.